src/types/Expr.d.ts

```diff
@@ -1,3 +1,5 @@
 export default class Expr {
   constructor(obj: object)
+
+  static toString(expr: Expr): string
```

Comment:
@BrunoQuaresma Is this a utility method to convert Expressions into Strings?

Reply:
Yes, it is.